include/stlink.h

```diff
@@ -15,6 +15,15 @@
 extern "C" {
 #endif
 
+#ifdef UNUSED
+#elif defined(__GNUC__)
+#define UNUSED(x)	UNUSED_ ## x __attribute__((unused))
+#elif defined(__LCLINT__)
+#define UNUSED(x)	/*@unused@*/ x
+#else
+#define UNUSED(x)	x
+#endif
+
```

Comment:
We should not do this, see my other comment. As this is highly compiler depended and I want in some time also support native visual studio compilation. Also when we really need to do this, sure not in the public scope.